src/reducers/pause.js

```diff
@@ -30,8 +30,8 @@ const State = makeRecord(({
   frames: undefined,
   selectedFrameId: undefined,
   loadedObjects: I.Map(),
-  shouldPauseOnExceptions: prefs.pauseOnExceptions,
-  shouldIgnoreCaughtExceptions: prefs.ignoreCaughtExceptions,
+  shouldPauseOnExceptions: false,
+  shouldIgnoreCaughtExceptions: false,
   expressions: I.List()
```

Comment:
On testing it seems like this change is causing it to lose state on pause.
See this
![foe](https://cloud.githubusercontent.com/assets/792924/22003782/4c439162-dc4e-11e6-8cf2-fa091240806f.gif)